### DISCUSSION TITLE
Lab Assignment #1

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/211A-ASIAAM-191A#3] [Assignments]
Paste your GitHub pages URL for Week 1 as a comment below to submit your [lab assignment #1](https://github.com/albertkun/211A-ASIAAM-191A/blob/master/Week_01/Lab/lab_assignment.md).

For example:
https://albertkun.github.io/211A-ASIAAM-191A-Assignments/week1/index.html

**Comment by cynthinee** [comment]
https://cynthinee.github.io/2021A-191A-Lab-Assignments/week1/

**Reply by cynthinee** [reply]
I repasted the code shared in class today and overrode my original coordinates, so that's been updated! :)

**Comment by edmondywen** [comment]
https://edmondywen.github.io/asiaam-191a/week1/index.html

**Comment by luigimagdamit** [comment]
https://luigimagdamitucla.github.io/211A-191A-Lab-Assignments/week1/

**Comment by hrfrahn** [comment]
https://hrfrahn.github.io/191A-Labs/week1/

**Comment by abi-200** [comment]
https://abi-200.github.io/191a-labs/week1/

**Comment by perezsara59** [comment]
https://perezsara59.github.io/211A-191A-Assignments/week1/

**Comment by emilysuh** [comment]
https://emilysuh.github.io/211A-191A-Lab-Assignment/week1/

**Comment by eandrusenki** [comment]
https://eandrusenki.github.io/AsAmS-191A-Assignments/Week1

**Comment by ramirezirvin** [comment]
https://ramirezirvin.github.io/191A-lab-assingments/week1/index.html

**Comment by ronnie-blanche** [comment]
http://10.1.10.179:61302/Week%201/
